### PR TITLE
Command plus random

### PR DESCRIPTION
**Pull request by OroArmor**
This will be after #355

### CODE REVIEW

File: mappings/net/minecraft/command/CommandSource.mapping [hunk 1]
```diff
@@ -28,23 +30,31 @@ CLASS net/minecraft/unmapped/C_sasfholk net/minecraft/command/CommandSource
 		ARG 0 string
 		ARG 1 candidates
 		ARG 2 builder
+		ARG 3 isPosition
 	METHOD m_jgssjzjq suggestColumnPositions (Ljava/lang/String;Ljava/util/Collection;Lcom/mojang/brigadier/suggestion/SuggestionsBuilder;Ljava/util/function/Predicate;)Ljava/util/concurrent/CompletableFuture;
 		ARG 0 string
 		ARG 1 relativePosition
 		ARG 2 builder
+		ARG 3 isPosition
```
Comment (eristhea) [inline mappings/net/minecraft/command/CommandSource.mapping:38]: this parameter doesn't really make sense to me without looking at the method's code: if I'm already passing in a parameter called `relativePosition`, why is there another called `isPosition`?

File: mappings/net/minecraft/command/argument/RegistryEntryOrTagArgument.mapping [hunk 2]
```diff
@@ -0,0 +1,39 @@
+CLASS net/minecraft/unmapped/C_bwiwqfgl net/minecraft/command/argument/RegistryEntryOrTagArgument
```
Comment (eristhea) [inline mappings/net/minecraft/command/argument/RegistryEntryOrTagArgument.mapping:1]: I don't like this name, but also can't think of a better way to express it
same applies to `RegistryKeyOrTagKeyResult`

**Review by eristhea — APPROVED** [review]
brilliant!

File: mappings/net/minecraft/client/gui/screen/narration/ScreenNarrator.mapping [hunk 3]
```diff
@@ -11,7 +11,7 @@ CLASS net/minecraft/unmapped/C_yviluxux net/minecraft/client/gui/screen/narratio
 		ARG 3 partIndex
 		ARG 4 message
 	METHOD m_ugxbjsky buildNarrations (Ljava/util/function/Consumer;)V
-		ARG 1 builderConsumer
+		ARG 1 consumer
```
Comment (OroArmor) [inline mappings/net/minecraft/client/gui/screen/narration/ScreenNarrator.mapping]:
```suggestion
		ARG 1 builderConsumer
```

File: mappings/net/minecraft/command/CommandSource.mapping (hunk 1 above)
Comment (OroArmor) [reply mappings/net/minecraft/command/CommandSource.mapping:38]: its a predicate

Comment (eristhea) [reply mappings/net/minecraft/command/CommandSource.mapping:38]: ah
I guess I just assumed boolean

**Review by EnnuiL — APPROVED** [review]

**Pull request merged**